### PR TITLE
Introduce schools data export rake tasks

### PR DESCRIPTION
**Pull request by mec**
So we can easily export csv files of eligble schools for both Student loans and Maths and physics.

Decided to add them to the existing `schools_data` namespace to keep things tidy.

Also took the opportunity to rename `eligible_for_tslr?` to `eligible_for_student_loans?` inline with the rest of the app as the rake task calls it.

### CODE REVIEW

**Comment by mec** [comment]
[Export Maths and Physics eligible schools list](https://trello.com/c/TL9RuzTQ/636-export-maths-and-physics-eligible-schools-list)

File: lib/tasks/schools_data.rake [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    export_school_data(:student_loans)
+  end
+
+  desc "Export schools eligible for Maths and physics to csv"
+  task "export:maths_and_physics": :environment do
+    export_school_data(:maths_and_physics)
+  end
+end
+
+def export_school_data(policy_name)
```
Comment (tekin) [inline lib/tasks/schools_data.rake]: Can we put this method in the `namespace` so it doesn't get globally defined?

File: lib/tasks/schools_data.rake [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  require "csv"
```
Comment (tekin) [inline lib/tasks/schools_data.rake]: It's more conventional to stick this at the top of the file rather than here.

Comment (mec) [reply lib/tasks/schools_data.rake]: bah! I moved it down! :(

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog]
 
 ## [Unreleased]
 
+- introduced rake tasks to export eligible schools to csv
```
Comment (tekin) [inline CHANGELOG.md]: Capitalise the `I` please.

**Pull request merged**